src/libraries/System.Transactions.Local/src/System/Transactions/TransactionManager.cs

```diff
@@ -391,6 +399,60 @@ public static TimeSpan MaximumTimeout
             }
         }
 
+        /// <summary>
+        /// Controls whether usage of System.Transactions APIs that require escalation to a distributed transaction will do so;
+        /// if your application requires distributed transaction, opt into using them by setting this to <see langword="true" />.
+        /// If set to <see langword="false" /> (the default), escalation to a distributed transaction will throw a <see cref="NotSupportedException" />.
+        /// </summary>
+#if WINDOWS
```

Comment:
Elsewhere in runtime we've used partial files for whole APIs whose implementations differ across platform.  If we use ifdefs elsewhere in System.Transactions.Local for this, it's ok.  We could also refactor it subsequently if we're trying to get this in asap.